### PR TITLE
HUBBLE-409 Update/delete airflow variable names

### PR DESCRIPTION
**Pull request by laysabit**
<details>
  <summary>PR Checklist</summary>

### PR Structure

- [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
- [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
      otherwise).
- [x] This PR's title starts with the jira ticket associated with the PR.

### Thoroughness

- [x] This PR adds tests for the most critical parts of the new functionality or fixes.
- [x] I've updated the README with the added features, breaking changes, new instructions on how to use the repository.

</details>

### What

README updated with new information.
Variable affinity is deleted.
Variable dbt_dataset was not deleted.

dbt_ was appended as prefix to these var names:

- internal_source_db
- internal_source_schema  
- public_source_db        
- public_source_schema    
- slack_elementary_channel
- elementary_secret     

### Why

dbt_dataset was not deleted because of its impact on the code:

how it should look without it:
how_should_look

how it looks without dbt_dataset:
how_it_looks

the error caused because of the lack of dbt_dataset:
error

### Known limitations

[N/A]

### CODE REVIEW

File: dags/dbt_stellar_marts_dag.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     },
     max_active_runs=3,
     catchup=True,
-    tags=["dbt-sdf-marts"],
+    tags=["dbt-stellar-marts"],  # trocar
```
Comment (sydneynotthecity) [inline dags/dbt_stellar_marts_dag.py:23]: nit: comment can be removed

File: dags/dbt_stellar_marts_dag.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 init_sentry()
 
 dag = DAG(
-    "dbt_sdf_marts",
+    "dbt_stellar_marts",
```
Comment (sydneynotthecity) [inline dags/dbt_stellar_marts_dag.py:13]: what was the reason for renaming this DAG?

**Review by sydneynotthecity — COMMENTED** [review]
Looks good, thanks for cleaning up the airflow variables. One question needs addressed and then I'll approve the PR

File: dags/dbt_stellar_marts_dag.py (hunk 2 above)
Comment (laysabit) [reply dags/dbt_stellar_marts_dag.py:13]: it was a demand by Simon: "Rename dag dbt_sdf_marts to like dbt_stellar_marts as it’s a public repo and not only SDF marts" [Source](https://stellarorg.atlassian.net/jira/software/c/projects/HUBBLE/boards/65?search=409&selectedIssue=HUBBLE-409)

File: dags/dbt_stellar_marts_dag.py (hunk 1 above)
Comment (laysabit) [reply dags/dbt_stellar_marts_dag.py]: Comment is deleted

**Comment by laysabit** [comment]
I changed the name of dbt_dataset, as it was asked by Eduardo Alves, to dbt_dataset_for_test, because it is only used in test environment, so it won't confuse the user.

**Review by sydneynotthecity — APPROVED** [review]
Thanks for the context, looks good!

**Pull request merged**